types/hooks/screens/useAnonymousChannelListScreenHook.types.ts

```diff
@@ -6,6 +6,7 @@ interface UseAnonymousChannelListScreenHook {
   goToChatScreen: (channel: ChannelList) => void;
   goToPostDetailScreen: (channel: ChannelList) => void;
   goToContactScreen: ({from}: ContactScreenPayload) => void;
+  goToCommunityScreen: (channel: ChannelList) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `goToCommunityScreen` function in the `UseAnonymousChannelListScreenHook` interface is a good change. It provides a clear way to navigate to the community screen, which seems to be a new feature introduced in this PR.
```diff
+  goToCommunityScreen: (channel: ChannelList) => void;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->